hack/e2e.sh

```diff
@@ -2,42 +2,14 @@
 
 set -eo pipefail
 
-echo "SUITE=>${SUITE}"
-if [[ -z "$RESOURCEGROUP" ]]; then
-    RESOURCEGROUP=$(awk '/^    resourceGroup:/ { print $2 }' <_data/containerservice.yaml)
-fi
-
 if [[ -n "$ARTIFACT_DIR" ]]; then
   ARTIFACT_FLAG="-artifact-dir=$ARTIFACT_DIR"
 fi
 
-if [[ -z "$SUITE" || "$SUITE" == "enduser" ]]; then
-  echo "Running end user e2e tests"
-  # Login as osadmin to simulate a regular user
-  password=$(awk '/^  endUserPasswd:/ { print $2 }' <_data/containerservice.yaml)
-  fqdn=$(awk '/^  fqdn:/ { print $2 }' <_data/containerservice.yaml)
-  export KUBECONFIG=_data/_out/enduser.kubeconfig
-  oc login $fqdn --username enduser --password $password --insecure-skip-tls-verify=true
-  go test ./test/e2e -timeout 20m -test.v -ginkgo.v -ginkgo.focus="\[EndUser\]" -ginkgo.noColor -ginkgo.randomizeAllSpecs -tags e2e -kubeconfig ../../_data/_out/enduser.kubeconfig "${ARTIFACT_FLAG:-}"
-  oc logout
+if [[ -n "$FOCUS" ]]; then
+    FOCUS="-ginkgo.focus=$FOCUS"
 fi
 
-if [[ -z "$SUITE" || "$SUITE" == "clusterreader" ]]; then
-  echo "Running azure cluster reader e2e tests"
-  (awk '/^  azureClusterReaderKubeconfig:/ { print $2 }' <_data/containerservice.yaml | base64 -d) > _data/_out/azure-cluster-reader.kubeconfig
-  go test ./test/e2e -timeout 20m -test.v -ginkgo.v -ginkgo.focus="\[AzureClusterReader\]" -ginkgo.noColor -ginkgo.randomizeAllSpecs -tags e2e -kubeconfig ../../_data/_out/azure-cluster-reader.kubeconfig
-fi
+go test ./test/e2e -timeout 20m -test.v -ginkgo.v "${FOCUS:-}" -ginkgo.noColor -tags e2e "${ARTIFACT_FLAG:-}"
```

Comment:
This timeout is not going to work if we want to run upgrades as part of the e2e tests.

Reply:
I ran into this as well

Reply:
#791